blazar/plugins/oshosts/host_plugin.py

```diff
@@ -211,6 +215,17 @@ def on_start(self, resource_id):
                             host_reservation['aggregate_id'], host)
                 raise
 
+        action = host_reservation.get('on_start', None)
```

Comment:
Why not use `'default'` as the default?

```suggestion
         action = host_reservation.get('on_start', 'default')
```